### ISSUE TITLE
EnergyPlus does not report QLat_flow and QPeo_flow

### ISSUE THREAD

**Issue by mwetter** [lbl-srg/modelica-buildings#2602]
EnergyPlus does not output the values for the latent heat gains of the zone and the heat emitted by people. See https://lbl-srg.github.io/soep/softwareArchitecture.html#coupling-of-the-envelope-model for the specification.

To reproduce, simulate 
```
Advanced.StoreProtectedVariables = true;
simulateModel("Buildings.ThermalZones.EnergyPlus.Examples.SingleFamilyHouse.Unconditioned", stopTime=432000, method="Cvode", tolerance=1e-06, resultFile="Unconditioned");
```
and look at `QPeo_flow` and `QLat_flow`.

This was tested on the master branch (commit 558ac77bfd98742220c872298e142e62c6fe755f).

**Comment by kbenne** [comment]
yes. I've been working on it. Couple of issues im addressing. 1. Back in version 0.1.0 we used a service called bintray to manage third party depndencies which is no longer available, so I had to back port the replacement. 2. I had to reimage the Mac CI machine. The builds are rerunning at this moment, so we'll see.
